nixos/modules/services/web-servers/unit/default.nix

```diff
@@ -0,0 +1,70 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.unit;
+
+  configFile = builtins.toFile "unit.json" ''
+    ${cfg.config}
+  '';
+
+in {
+  options = {
+    services.unit = {
+      enable = mkEnableOption "Unit App Server";
+      package = mkOption {
+        type = types.package;
+        default = pkgs.unit;
+        description = "Unit package to use.";
```

Comment:
Do you need this option? Because if you don't, I prefer to not have such an option. There are mixed opinions on this though, see https://github.com/NixOS/nixpkgs/issues/50476.

Reply:
Yes? need this option. Me use custom package unit with php56 - https://github.com/Izorkin/nur-packages/blob/master/pkgs/servers/unit/default.nix